src/compiler/config/test/validate-namespace.spec.ts

```diff
@@ -3,90 +3,77 @@ import type * as d from '@stencil/core/declarations';
 import { validateNamespace } from '../validate-namespace';
 
 describe('validateNamespace', () => {
-  let config: d.Config;
   const diagnostics: d.Diagnostic[] = [];
+
   beforeEach(() => {
-    config = {};
     diagnostics.length = 0;
   });
 
   it('should not allow special characters in namespace', () => {
-    config.namespace = 'My/Namespace';
-    validateNamespace(config, diagnostics);
+    validateNamespace('My/Namespace', undefined, diagnostics);
     expect(diagnostics).toHaveLength(1);
 
     diagnostics.length = 0;
-    config.namespace = 'My%20Namespace';
-    validateNamespace(config, diagnostics);
+    validateNamespace('My%20Namespace', undefined, diagnostics);
     expect(diagnostics).toHaveLength(1);
 
     diagnostics.length = 0;
```

Comment:
Can you do me a favor and put a task in the backlog to improve theses tests? Specifically, I'd like us to test the contents of the diagnostic, rather than the fact it exists.